src/h5web/visualizer/Visualizer.tsx

```diff
@@ -43,7 +45,10 @@ function Visualizer<T extends VisDef>(props: Props<T>) {
         <div className={styles.displayArea}>
           <ErrorBoundary
             resetKeys={[entity.path]}
-            FallbackComponent={ErrorMessage}
+            FallbackComponent={ErrorFallback}
+            onReset={() => {
+              valuesStore.evict({ path: entity.path });
```

Comment:
For other visualizations, I just need to pass the entity's `path` since there's no slice selection.